pyproject.toml

```diff
@@ -16,6 +16,9 @@ include_trailing_comma = true
 [tool.jupyter-releaser.options]
 check-imports = ["quetz"]
 
+[tool.check-wheel-contents]
+ignore = ["W004"]
```

Comment:
It's recommended to ignore this check for Alembic migration files. Ignoring it on a file level is not yet supported ([details](https://github.com/jwodder/check-wheel-contents#w004--module-is-not-located-at-importable-path)).